### PR TITLE
feat: set real data to `TrendOfPassAndFailFindings` widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
스크린샷 2023-06-13 오후 9 57 18

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue [hunk 1]
```diff
@@ -68,14 +74,20 @@ import { useWidgetFrameProps } from '@/services/dashboards/widgets/_hooks/use-wi
...
+    value: Array<{ key: string, value: number }>;
+}
+interface TableDataModel extends CloudServiceStatsModel {
+    value: Array<{ date: string; value: number }>;
+}
 interface FullData {
-    results: Data;
-    more: boolean;
+    chartData?: { results: ChartDataModel[] };
+    tableData?: { more: boolean, results: TableDataModel[] };
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue:88]: praise: wow

File: apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue [hunk 2]
```diff
@@ -89,27 +101,34 @@ const { colorSet } = useWidgetColorSet({
...
 const state = reactive({
     ...toRefs(useWidgetState<FullData>(props)),
-    // chartData: computed<XYChartData[]>(() => getRefinedXYChartData(state.data?.results, state.groupBy)),
-    chartData: computed(() => ([
-        { date: '2023-02', PASS: random(300, 500), FAIL: random(0, 100) },
-        { date: '2023-03', PASS: random(300, 500), FAIL: random(0, 100) },
-        { date: '2023-04', PASS: random(300, 500), FAIL: random(0, 100) },
-        { date: '2023-05', PASS: random(300, 500), FAIL: random(0, 100) },
-    ])),
+    groupByKey: computed<string>(() => {
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue:104]: suggestion: return type must be `string|undefined`

**Review by WANZARGEN — APPROVED** [review]
Wow.. How complex the requirement is...!!!!! 😭

File: apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue [hunk 3]
```diff
@@ -161,16 +245,40 @@ const drawChart = (chartData: XYChartData[]) => {
...
+                });
+            });
+            let _text = `Total: [bold; fontSize: 14px]${totalValue}[/]`;
+            seriesList.forEach((s) => {
+                const rate = Math.round((s.value / totalValue) * 100);
+                _text += `\n[${s.color}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.name}:[/] [bold; fontSize: 14px]${s.value}[/] (${rate}%)`;
+            });
+            return _text;
+        });
+        series.set('tooltip', tooltip);
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue:269]: description: Tooltip in this widget is little different from other widgets....🥲 So I custom here

**Review by sulmoJ — APPROVED** [review]
LGTM!
I checked only code convention!

**Pull request merged**